src/types/react-relay-network-modern.d.ts

```diff
@@ -0,0 +1,311 @@
+/* tslint:disable */
+
+declare module "react-relay-network-modern/es" {
```

Comment:
Can we consider submitting these to @types? With a `TODO` to replace them in the future?

Reply:
Of course